cmd/dockerd/daemon.go

```diff
@@ -92,7 +90,7 @@ func (cli *DaemonCli) start(opts *daemonOptions) (err error) {
 
 	if opts.Validate {
 		// If config wasn't OK we wouldn't have made it this far.
-		fmt.Fprintln(os.Stderr, "configuration OK")
+		_, _ = fmt.Fprintln(os.Stderr, "configuration OK")
```

Comment:
:raised_eyebrow:

Reply:
Got fed up with my IDE complaining about the unhandled error 😂